### ISSUE TITLE
Button formatting with il-formatted tag

### ISSUE THREAD

**Issue by nbaxley-uiuc** [ATLAS-Illinois/Illinois-Framework-Theme#554]
There are several places where adding the il-formatted tag to the main div tag causes issues with button formatting.  Unfortunately the buttons are applied in some very specific places that are going to require some more intricate fixes.

## Hero
Color combinations are not being honored.  These are the four combinations but I'm not sure if we're using the il-hero component incorrectly or something else.

- [ ] Illini Blue background with orange buttons
- [ ] Image background with orange buttons
- [ ] Illini Orange background with blue buttons
- [ ] Image background with blue buttons

## Call to Action
Similar issues with button/background combinations.  We are using the il-button class this time, but with custom background/button styling rules which are being overridden.  Background options to handle:

- [ ] Blue Gradient
- [ ] Orange Gradient
- [ ] Illini Blue
- [ ] Illini Orange
- [ ] Gray
- [ ] White

## Feature Split/Feature Video/Intro
Each of these paragraphs are using the il-button class but no coloring classes, instead they are relying on the background color to set the button color which is causing issues with il-fromatted.  Here are the background options to be addressed:

- [ ] Illini Blue
- [ ] Blue Gradient
- [ ] Orange Gradient
- [ ] White

**Comment by nbaxley-uiuc** [comment]
@mwllr and @magjarvis I'm not sure who created the CSS classes for our background/button combos.  They are great in what they do, but they don't use the button color classes from campus.  We either need to change to using those (and I guess add more background logic to the templates) or make the CSS rules explicit enough that they can override the campus classes.  This is pretty important one if we want to add il-formatted to the next release.

**Comment by wrg20** [comment]
@nbaxley-uiuc when the buttons were created there was no il-formatted classes when backgrounds were being considered.  That has just recently been looked at by design.  So using il-formatted will require changing how our backgrounds work on all of the paragraphs.

**Comment by nbaxley-uiuc** [comment]
After Melissa and I talked yesterday morning we realized that we had to put the il-formatted on the individual paragraphs instead of the entire content area.  The Hero web component, for example, explicitly expects il-formatted to NOT be present.  I pushed up a change last night to add the il-formatted to most paragraphs and the body section on other CTs.  I left it off of the paragraph types that had button/background issues.
